### ISSUE TITLE
Does not raise errors when referencing non-existing properties

### ISSUE THREAD

**Issue by agross** [Dzoukr/Fue#14]
```
With placeholders: {{{ config.Foo }}} and {{{ config.Bar }}}
{{{ config.DoesNotExist }}}
```

`config`s type does not contain a definition for `DoesNotExist`, yet rendering is successful. I wonder if this is intentional.

**Comment by Dzoukr** [comment]
Yeah, the value is just not replaced at all. It was designed to be not so intrusive (IMHO it is better have all rendered except one than have full failure), but I am gathering improvements ideas for next major version so please if you see any area for improvement, keep it coming.

**Comment by agross** [comment]
> It was designed to be not so intrusive 

I'd say this makes Fue unreliable. I'm currently investigating if I could use Fue to template config files. It would be unfortunate to create configs that won't run in production because someone msityped a key. See, it already happened ;-)

Here's what I have in mind: https://github.com/agross/fake-config-templates/blob/master/build.fsx#L31-L69

**Comment by Dzoukr** [comment]
I see. Hmmm. I'll keep that in mind for next version, maybe this is good intention, but bad behavior. :)
